test-utils/test-utils-core/src/main/java/uk/gov/justice/services/test/utils/core/http/PollingRequestParams.java

```diff
@@ -17,52 +15,73 @@
     private final String mediaType;
     private final MultivaluedMap<String, Object> headers;
 
-    private final Predicate<Response> responseCondition;
-    private final Predicate<String> resultCondition;
+    private final Predicate<String> resposeBodyCondition;
     private final long delayInMillis;
     private final int retryCount;
+    private final Optional<Integer> expectedStatus;
```

Comment:
Can this be updated to use a Status rather than an Integer.
javax.ws.rs.core.Response.Status